.vscode/cspell.json

```diff
@@ -430,7 +430,9 @@
     "dacl",
     "whls",
     "aiter",
-    "solft"
+    "solft",
+    "amltmp",
```

Comment:
what is this for?

Reply:
this is the word list to exclude from spell check